modules/advertlyBidAdapter.js

```diff
@@ -0,0 +1,129 @@
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { config } from '../src/config';
+import * as utils from '../src/utils';
+import {BANNER, VIDEO} from '../src/mediaTypes';
+import { ajax } from '../src/ajax';
+import {Renderer} from '../src/Renderer';
+
+const SUPPORTED_AD_TYPES = [BANNER, VIDEO];
+const BIDDER_CODE = 'advertly';
+const DOMAIN = 'http://api.advertly.com/';
```

Comment:
Calls should be made using HTTPS

Reply:
Does your server support HTTPS? It needs to per the Prebid docs:

http://prebid.org/dev-docs/bidder-adaptor.html#bidder-adaptor-Required-Adapter-Conventions

Reply:
Okay. We will upgrade our sever with SSL (HTTPS)

Reply:
We have upgrade our sever with HTTPS. And HTTPS changed in library files.

Reply:
Hi @msm0504 
We have upgrade the server to HTTPS. can you check it?

Reply:
Looks good. Thanks